BCC/src/main/java/BNU/logic/TeacherReviewController.java

```diff
@@ -91,8 +91,10 @@ public void actionPerformed(ActionEvent e) {
 		if(e.getActionCommand() == "teacher_review:back"){
 			System.out.println("teacher_review:back button pressed");
 			WindowBuilder.loadPage(new MainController());
-		
-	}
+		}else if(e.getActionCommand() == "teacher_review:add"){
+			System.out.println("teacher_review:add Review button pressed");
```

Comment:
was there supposed to be a button on the review page? i dont see one if there is...